src/main/webapp/js/playGame.js

```diff
@@ -1,4 +1,5 @@
 var currGameData = new Progress([]);
+var minimap;
```

Comment:
When is it global? Does it need to be global?

Reply:
Minimap is no longer global, thanks.